native-maven-plugin/src/main/java/org/graalvm/buildtools/utils/NativeImageConfigurationUtils.java

```diff
@@ -57,7 +56,7 @@
 /**
  * Utility class containing various native-image and JVM related methods.
  */
-public abstract class Utils implements SharedConstants {
+public abstract class NativeImageConfigurationUtils implements SharedConstants {
```

Comment:
Worth to test that with Spring/Micronaut to be sure it does not break current integrations (we had that kind of regression in the past).

Reply:
@alvarosanchez could you please test this PR with Micronaut?
@bclozel could you please test this PR with Spring?

Reply:
Compilation of Boot seems ok.

@alvarosanchez Could you please check on Micronaut side?

Reply:
@sdeleuze as per your last review comment, it doesn't really work, does it?

Reply:
I meant compilation of Boot itself depending on the NBT snapshot of this PR was fine, usage in a project was broken but now fixed I think and likely unrelated to this class rename.

Reply:
I tested this PR with Micronaut too, doesn't seem to break anything